src/document/DocumentModel.tsx

```diff
@@ -89,7 +88,6 @@ export const DocumentStore = types
     serializeChor(): Project {
       return {
         name: self.name,
-        version: PROJECT_SCHEMA_VERSION,
```

Comment:
just change this value to a call from the backend or you can even lazily load the value at bootup and store it for the lifetime of the process.